examples/meta/src/evaluation/multiclassovrevaluation.sg.in

```diff
@@ -0,0 +1,29 @@
+File traindat = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
```

Comment:
sorry I should have mentioned this but the filename here should also be more readable
"multiclass_ovr_evaluation"